### PR TITLE
validate both noopener and noreferrer by extending the rule config

### PR DESCRIPTION
**Pull request by ingride**
Fix for #187 .

This PR extends the configuration options for `link-rel-noopener` to allow enforcing validation for both `noopener` **and** `noreferrer` on  rule.

Backward compatibility is preserved by maintaining the boolean values `true` to enable and `false` to disable. 

To configure the new validation

``` javascript
module.exports = {
  rules: {
    'link-rel-noopener': [true, 'strict']
  }
}; 
```

cc @paddyobrien  and @GavinJoyce

### CODE REVIEW

File: lib/rules/lint-link-rel-noopener.js [hunk 1]
```diff
@@ -18,9 +18,45 @@ var buildPlugin = require('./base');
...
+
 module.exports = function(addonContext) {
   var LinkRelNoopener = buildPlugin(addonContext, 'link-rel-noopener');
 
+  LinkRelNoopener.prototype.parseConfig = function(config) {
+    var configType = typeof config;
+
+    var errorMessage = 'The bare-strings rule accepts one of the following values.\n ' +
+      '  * boolean - `true` to enable / `false` to disable\n' +
+      '  * array -- [`true`, `strict`] to enable validation for both noopener AND noreferrer\n' +
```
Comment (rwjblue) [inline lib/rules/lint-link-rel-noopener.js]: Can we swap to `true`, `false`, and `'strict'` (e.g. use a single string for `"strict"` mode).

Comment (ingride) [inline lib/rules/lint-link-rel-noopener.js]: done

Comment (rwjblue) [inline lib/rules/lint-link-rel-noopener.js]: Implementation looks good, but this error message needs to be updated I believe.

**Comment by rwjblue** [comment]
Thank you!!

**Pull request merged**